README.md

```diff
@@ -99,13 +85,13 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
-[OPTIONAL] I think removing this kind of text will look more professional as this text refers to the writer of the readme and not to the visitor.